content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -43,6 +43,9 @@ Istio control plane, Envoy can then communicate with the **single**
 Istio control plane and form a mesh network across multiple Kubernetes
 clusters.
 
+This guide describes how to install a multicluster Istio topology using the 
+manifests and helm charts provided within the Istio repository.  
```

Comment:
since you have to rev for Tim's comment, might capitalize Helm.

Reply:
will do